fedimint-cli/src/client.rs

```diff
@@ -349,6 +355,51 @@ pub async fn handle_ng_command(
                 "secret": hex_secret,
             }))
         }
+        ClientCmd::ListOperations { limit } => {
+            #[derive(Serialize)]
+            struct OperationOutput {
+                id: OperationId,
+                creation_time: String,
+                operation_kind: String,
+                operation_meta: serde_json::Value,
+                #[serde(skip_serializing_if = "Option::is_none")]
+                outcome: Option<serde_json::Value>,
+            }
+
+            const ISO8601_CONFIG: iso8601::EncodedConfig = iso8601::Config::DEFAULT
+                .set_formatted_components(iso8601::FormattedComponents::DateTime)
+                .encode();
+            let operations = client
+                .operation_log()
+                .list_operations(limit, None)
+                .await
+                .into_iter()
+                .map(|(k, v)| {
+                    //
```

Comment:
`//` intentional (for formatting maybe?) or leftover?

Reply:
fixed